source/openpower/keys.rst

```diff
@@ -0,0 +1,160 @@
+.. _keys:
+
+SSH Key Behavior and Management on OpenPOWER
+============================================
+
+These are findings of how SSH Keys behave in various conditions on OSL's POWERDev OpenStack cluster, and how to manage these keys.
+
+.. note::
+
+  For beginners: a good tutorial on SSH keys and SSH in general `here`__.
+  
+  And a quicker, more advanced and jargon-full tutorial `here`__. 
+  
+  *Probably best to read both, even if you're advanced. There are some extremely helpful tips and tricks in both of these.*
+  
+.. __: https://docstore.mik.ua/orelly/networking_2ndEd/ssh/ch01_01.htm
+
+.. __: https://www.digitalocean.com/community/tutorials/understanding-the-ssh-encryption-and-connection-process
+
+Accessing an Instance Using an SSH Key
+--------------------------------------
+
+On a simple level, instances can be accessed using ``ssh``::
+
+  ~$ ssh <image_name>@<instance_IP_address>
+
+*Example:* ``~$ ssh ubuntu@140.211.123.456``
+
+.. note::
+
+  <image_name> changes after rebuilding an instance with a different OS. Ubuntu machines -> ``ubuntu@<IP>`` CentOS machines -> ``centos@<IP>``, Debian machines -> ``debian@<IP>``, Fedora machines -> ``fedora@<IP>``, etc...
+
+Doing so will prompt the server and client to establish a secure connection, then the server authenticates the requesting client.
+
+If you have the correct credentials (the desired SSH private key), then you will be granted access to the server.
+
+.. note::
+
+  This description is very brief and not detailed enough to fully eclipse how SSH keys and the Diffie-Hellman exchange works. A more descriptive overview is linked in the section `above.`__
```

Comment:
Instead of saying `fully eclipse`, it should probably just be `fully explain`. Eclipse doesn't make that much sense in this context.